bluesky_queueserver/manager/profile_ops.py

```diff
@@ -1350,7 +1350,7 @@ def load_profile_collection_from_ipython(path=None):
         load_profile_collection_from_ipython()
     """
     ip = get_ipython()  # noqa F821
-    for f in sorted(glob.glob("[0-9][0-9]*.py")):
+    for f in sorted(glob.glob("*.py")):
```

Comment:
IPython also executes files named `*.ipy`. These can be used to invoke non-Python syntax, such as magics. This is the relevant method in IPython, for comparison. https://github.com/ipython/ipython/blob/3587f5bb6c8570e7bbb06cf5f7e3bc9b9467355a/IPython/core/shellapp.py#L403

Reply:
Added `*.ipy` extension in all instances.